Tests/GRPCTests/TLSVerificationHandlerTests.swift

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2019, gRPC Authors All rights reserved.
```

Comment:
```suggestion
 * Copyright 2020, gRPC Authors All rights reserved.
```

Reply:
Oh dear! Sorry, I guess I need to get used to the idea :D